docs/helm-broker/03-03-create-addons-repo.md

```diff
@@ -3,8 +3,45 @@ title: Create addons repository
 type: Details
 ---
 
-The repository in which you create your own addons must have a specific structure and be exposed as an HTTPS server so that the Helm Broker can fetch addons from it. Your remote addons repository can contain many addons, each one compressed to the `.tgz` format and defined in `index.yaml` files. Depending on your needs and preferences, you can create one or more `index.yaml` files to categorize your addons. The repository structure looks as follows:
+The repository in which you create your own addons must contain at least one `index.yaml` file and have a specific structure, depending on the type of server that exposes your addons.
 
+## The index yaml file
+
+Your remote addons repository can contain many addons defined in index `.yaml` files. Depending on your needs and preferences, you can create one or more index yaml files to categorize your addons. In the `index.yaml` file, provide an entry for every single addon from your addons repository. The `index.yaml` file must have the following structure:
+```
+apiVersion: v1
+entries:
+  {addon_name}:
+    - name: {addon_name}
+      description: {addon_description}
+      version: {addon_version}
+```
+
+See the example:
+```yaml
+apiVersion: v1
+entries:
+  redis:
+    - name: redis
+      description: Redis service
+      version: 0.0.1
+```
+
+>**NOTE:** You must place your addons in the same directory where the `index.yaml` file is stored.
+
+## Supported protocols
+
+Expose your addons repository through a remote server so that you can provide URLs in the [AddonsConfiguration](#custom-resource-addonsconfiguration) (AC) or [ClusterAddonsConfiguration](#custom-resource-clusteraddonsconfiguration) (CAC) custom resources. The Helm Broker supports exposing addons using the following protocols:
```

Comment:
```suggestion
Expose your addons repository so that you can provide URLs in the [AddonsConfiguration](#custom-resource-addonsconfiguration) (AC) or [ClusterAddonsConfiguration](#custom-resource-clusteraddonsconfiguration) (CAC) custom resources. The Helm Broker supports exposing addons using the following protocols:
```